Public/Src/FrontEnd/Script/Ambients/Transformers/AmbientTransformer.Process.cs

```diff
@@ -545,7 +547,10 @@ private void ProcessExecuteArguments(Context context, ObjectLiteral obj, Process
             var errorRegex = Converter.ExtractString(obj, m_executeErrorRegex, allowUndefined: true);
             if (errorRegex != null)
             {
-                processBuilder.ErrorRegex = new RegexDescriptor(StringId.Create(context.StringTable, errorRegex), RegexOptions.None);
+                var enableMultiLine = Converter.ExtractOptionalBoolean(obj, m_executeErrorRegexEnableMultiLineMatches);
+                processBuilder.ErrorRegex = new RegexDescriptor(
+                    StringId.Create(context.StringTable, errorRegex),
+                    enableMultiLine == true ? RegexOptions.Singleline : RegexOptions.None);

```

Comment:
>enableMultiLine == true ? RegexOptions.Singleline : RegexOptions.None [](start = 20, length = 69)

This is splitting hairs a bit but our configuration option to process stdout/stedrr line by line isn't exactly the same as someone setting RegexOptions.Singleline. RegexOptions.Singleline is about letting regex matches span line breaks, essentially letting "." match all characters including \n instead of breaking on it. 
[https://docs.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-options?view=netframework-4.8#Singleline](https://docs.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-options?view=netframework-4.8#Singleline)

Had someone set RegexOptions.Singleline today we wouldn't have honored it. I guess you're using that as a mechanism to thread this context through. I'm wondering if it would be more clear if we added a bool to RegexDescription to more specifically denote that the regex should be applied line by line vs. against the entire stream